src/index.html

```diff
@@ -5,8 +5,474 @@
+              <p class="card__price-price">
+                Price:
+              </p>
+
+              <p class="card__price-value">
+                $2,199
+              </p>
+            </div>
+
+            <a
+              class="card__button"
+              href="#buy"
+              data-qa="card-hover">
+                Buy
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac picture"
+          >
+
+          <div class="card__content">
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="card__code">
+              Product code: 195434
+            </p>
+
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-price">
+                Price:
+              </p>
+
+              <p class="card__price-value">
+                $2,199
+              </p>
+            </div>
+
+            <a
+              class="card__button"
+              href="#buy"
+              data-qa="hover">
+                Buy
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac picture"
+          >
+
+          <div class="card__content">
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="card__code">
+              Product code: 195434
+            </p>
+
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-price">
+                Price:
+              </p>
+
+              <p class="card__price-value">
+                $2,199
+              </p>
+            </div>
+
+            <a
+              class="card__button"
+              href="#buy"
+              data-qa="hover">
+                Buy
+            </a>
```

Comment:
Please check yourself and fix all cases
```suggestion
            <a
              class="card__button"
              href="#buy"
              data-qa="hover"
            >
              Buy
            </a>
```